.releaserc.yaml

```
@@ -0,0 +1,33 @@
+plugins:
+  - - '@semantic-release/commit-analyzer'
+    - preset: conventionalcommits
+  - - '@semantic-release/release-notes-generator'
+    - preset: conventionalcommits
+  - semantic-release-obsidian-plugin
+  - - '@semantic-release/github'
+    - assets:
+        - path: main.js
+        - path: manifest.json
+        - path: src/ui/styles/styles.css
+  # Uncomment the following block if semantic release *should* commit the 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)